instance/config.py

```diff
@@ -1,3 +0,0 @@
-SECRET_KEY = '21%hbba7&njb#ggj@'
-JWT_SECRET_KEY = 'jwt_secret_string'
-SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:f1730043@localhost/weconnect'
```

Comment:
Why did you decide to do away with `instance/config` 🤔 
Have a look at this [link](http://exploreflask.com/en/latest/organizing.html) on how to organise your work and what goes where.

Reply:
I don't understand, can you clarify why I should push `instance/config` to remote?
I decided to include it in .gitignore because it was causing errors on TravisCI.

Reply:
Essentially `config.py` is used for environment variables for the whole application whereas `instance/config.py` is used to specify different variables for the many environment a project has i.e. `development`, `production` and `staging`

Reply:
Okay 
Now I get it, let work on the proper organization for configuration files.
Thanks a lot for the info.